test/nextjs.test.js

```diff
@@ -958,17 +1002,19 @@ if (major >= 18) {
             }
         });
 
-        it("should go to next error handler when withSession is called without core", async function () {
+        it.skip("should go to next error handler when withSession is called without core", async function () {
```

Comment:
Please add a comment about why we are skipping this and/or create an issue about re-enabling